lib/modules/hosts/strawpoll.js

```diff
@@ -0,0 +1,11 @@
+addLibrary('mediaHosts', 'strawpoll', {
+	domains: ['strawpoll.me'],
+	detect: href => (/^https?:\/\/(?:www\.)?strawpoll\.me\/(embed_[0-9]\/)?([0-9]*)/i).exec(href),
+	handleLink(elem, [, , uid]) {
```

Comment:
If you make `(embed_[0-9]\/)` a non-capturing group (`(?:embed_[0-9]\/)`), then this can just be `[, uid]`